testsuite/pyunit/dom/Sanity.py

```diff
@@ -46,10 +46,9 @@
 _TESTSUITE_ROOT = Path(__file__).parent.parent.parent.resolve()
 _GHDL_ROOT = _TESTSUITE_ROOT.parent
 
+# design = Design()
 
-design = Design()
-
-@mark.parametrize("file", [str(f.relative_to(_TESTSUITE_ROOT)) for f in _TESTSUITE_ROOT.glob("sanity/**/*.vhdl")])
+@mark.parametrize("file", [str(f.relative_to(_GHDL_ROOT)) for f in _TESTSUITE_ROOT.glob("sanity/**/*.vhdl")])
```

Comment:
This corrects how sanity checks are found and analyzed.

Before, sanity checks where empty, so all tests passed.

Now we get:

> FAILED pyunit/dom/Sanity.py::test_AllVHDLSources[testsuite/sanity/004all08/ams08.vhdl]
> FAILED pyunit/dom/Sanity.py::test_AllVHDLSources[testsuite/sanity/004all08/all08.vhdl]
> FAILED pyunit/dom/Sanity.py::test_AllVHDLSources[testsuite/sanity/002hello2008/err93.vhdl]
> FAILED pyunit/dom/Sanity.py::test_AllVHDLSources[testsuite/sanity/002hello2008/hello.vhdl]
> FAILED pyunit/dom/Sanity.py::test_AllVHDLSources[testsuite/sanity/005examples/adder_tb.vhdl]
> FAILED pyunit/dom/Sanity.py::test_AllVHDLSources[testsuite/sanity/005examples/adder.vhdl]
> FAILED pyunit/dom/Sanity.py::test_AllVHDLSources[testsuite/sanity/005examples/heartbeat.vhdl]
> FAILED pyunit/dom/Sanity.py::test_AllVHDLSources[testsuite/sanity/005examples/hello.vhdl]
> FAILED pyunit/dom/Sanity.py::test_AllVHDLSources[testsuite/sanity/001hello87/err87.vhdl]
> FAILED pyunit/dom/Sanity.py::test_AllVHDLSources[testsuite/sanity/001hello87/hello.vhdl]
> FAILED pyunit/dom/Sanity.py::test_AllVHDLSources[testsuite/sanity/006upf/test.vhdl]
> FAILED pyunit/dom/Sanity.py::test_AllVHDLSources[testsuite/sanity/000hello/hello.vhdl]